monitoring/prober/common.py

```diff
@@ -21,3 +23,25 @@
 
 GEO_POLYGON_STRING = ','.join(
     '{},{}'.format(x['lat'], x['lng']) for x in VERTICES)
+
+HUGE_VERTICES = [
+    {
+        'lat': 130,
```

Comment:
Did this work before and does it work now?  130 degrees is an invalid latitude so it seems like we should return 400 regardless of anything else.  Presumably the latitude and longitude values are just swapped in the test data.  Same issue with VERTICES.

Reply:
This worked. I found it weird too that all the vertices used throughout all the test are not on the surface of the earth, but I thought it might have been intentional. If you think we should go with a proper one then I can change it

Reply:
Yes, there are two issues: with lat=130, this test should return 400.  So if it returned 413 with lat=130, we should add another test case (probably in a different PR) that reveals that flaw (the flaw of not returning 400 for invalid latitudes).  Also, we should fix VERTICES and HUGE_VERTICES to have valid lat/lngs (probably by switching the lat and lng labels) to make these tests behave as intended.

Reply:
Other tests that uses  VERTICES that create areas returns 200, that means there is something wrong with the happy case too, we should not be able to create ISA or subs that is not on the surface of the earth